src/org/traccar/reports/ReportUtils.java

```diff
@@ -30,6 +30,8 @@
     private ReportUtils() {
     }
 
+    public static final String IGNORE_ODOMETER = "report.ignoreOdometer";
```

Comment:
As soon as we have a separate function to get config parameter, I guess we won't need a constant for it because it will only be in one/two places.